.gitignore

```diff
@@ -4,6 +4,7 @@ packages/*/docs
 .node-version
 .parcel-cache
 dist*/
+docs
```

Comment:
Where did the commit go which fixed this? I know I fixed this at some point, but this is not the correct (or complete) fix. The folder in the `./website/docs` gets ignored this way, which is incorrect.

Reply:
See #563